src/main/java/frc/robot/subsystems/Limelight.java

```diff
@@ -19,11 +25,27 @@ public void periodic() {
     final double x = tx.getDouble(0.0);
     final double y = ty.getDouble(0.0);
     final double area = ta.getDouble(0.0);
+    final String stream, cam;
+
+    if(getStream() == Constants.StreamMode.PIP_MAIN)
+      stream = "Main";
+    else
+      stream = "Secondary";
```

Comment:
Even though not strictly necessary let's add `{` `}`, makes it easier to extend going forward.